keyboards/varanidae/info.json

```diff
@@ -0,0 +1,156 @@
+                { "label":"K042", "matrix": [2, 3], "x": 2, "y": 2.25, "w": 1 },
+                { "label":"K043", "matrix": [2, 4], "x": 3, "y": 2.25, "w": 1 },
+                { "label":"K044", "matrix": [2, 5], "x": 4, "y": 2.25, "w": 1 },
+                { "label":"K045", "matrix": [2, 6], "x": 5.25, "y": 2.25, "w": 1.5 },
+                { "label":"K046", "matrix": [2, 7], "x": 6.75, "y": 2.25, "w": 1 },
+                { "label":"K047", "matrix": [2, 8], "x": 7.75, "y": 2.25, "w": 1 },
+                { "label":"K048", "matrix": [2, 9], "x": 8.75, "y": 2.25, "w": 1 },
+                { "label":"K049", "matrix": [2, 10], "x": 9.75, "y": 2.25, "w": 1 },
+                { "label":"K050", "matrix": [7, 6], "x": 10.75, "y": 2.25, "w": 1 },
+                { "label":"K051", "matrix": [7, 7], "x": 11.75, "y": 2.25, "w": 1 },
+                { "label":"K052", "matrix": [7, 8], "x": 12.75, "y": 2.25, "w": 1 },
+                { "label":"K053", "matrix": [7, 9], "x": 13.75, "y": 2.25, "w": 1 },
+                { "label":"K054", "matrix": [7, 10], "x": 14.75, "y": 2.25, "w": 1 },
+                { "label":"K055", "matrix": [9, 1], "x": 15.75, "y": 2.25, "w": 1 },
+                { "label":"K056", "matrix": [9, 2], "x": 16.75, "y": 2.25, "w": 1 },
+                { "label":"K057", "matrix": [9, 3], "x": 17.75, "y": 2.25, "w": 1 },
+                { "label":"K058", "matrix": [9, 4], "x": 18.75, "y": 2.25, "w": 1.5 },
+
+                { "label":"K059", "matrix": [3, 1], "x": 0, "y": 3.25, "w": 1 },
+                { "label":"K060", "matrix": [3, 2], "x": 1, "y": 3.25, "w": 1 },
+                { "label":"K061", "matrix": [3, 3], "x": 2, "y": 3.25, "w": 1 },
+                { "label":"K062", "matrix": [3, 4], "x": 3, "y": 3.25, "w": 1 },
+                { "label":"K063", "matrix": [3, 5], "x": 4, "y": 3.25, "w": 1 },
+                { "label":"K064", "matrix": [3, 6], "x": 5.25, "y": 3.25, "w": 1.75 },
+                { "label":"K065", "matrix": [3, 7], "x": 7, "y": 3.25, "w": 1 },
+                { "label":"K066", "matrix": [3, 8], "x": 8, "y": 3.25, "w": 1 },
+                { "label":"K067", "matrix": [3, 9], "x": 9, "y": 3.25, "w": 1 },
+                { "label":"K068", "matrix": [3, 10], "x": 10, "y": 3.25, "w": 1 },
+                { "label":"K069", "matrix": [8, 6], "x": 11, "y": 3.25, "w": 1 },
+                { "label":"K070", "matrix": [8, 7], "x": 12, "y": 3.25, "w": 1 },
+                { "label":"K071", "matrix": [8, 8], "x": 13, "y": 3.25, "w": 1 },
+                { "label":"K072", "matrix": [8, 9], "x": 14, "y": 3.25, "w": 1 },
+                { "label":"K073", "matrix": [8, 10], "x": 15, "y": 3.25, "w": 1 },
+                { "label":"K074", "matrix": [10, 1], "x": 16, "y": 3.25, "w": 1 },
+                { "label":"K075", "matrix": [10, 2], "x": 17, "y": 3.25, "w": 1 },
+                { "label":"K076", "matrix": [10, 3], "x": 18, "y": 3.25, "w": 2.25 },
```

Comment:
```suggestion
                { "label":"K058", "matrix": [9, 4], "x": 18, "y": 3.25, "w": 2.25 },
```

Reply:
As described in https://github.com/qmk/qmk_firmware/pull/21340#discussion_r1241316148, perhaps this issue is a wrong shape of LAYOUT and not a wrong matrix.